recipes/libultrahdr/all/patches/1.3.0_cmake-portability.patch

```diff
@@ -0,0 +1,163 @@
+-        INSTALL_COMMAND ""
+-    )
+-  else()
+-    ExternalProject_Add(${JPEGTURBO_TARGET_NAME}
+-        GIT_REPOSITORY https://github.com/libjpeg-turbo/libjpeg-turbo.git
+-        GIT_TAG 3.0.1
+-        PREFIX ${JPEGTURBO_PREFIX_DIR}
+-        SOURCE_DIR ${JPEGTURBO_SOURCE_DIR}
+-        BINARY_DIR ${JPEGTURBO_BINARY_DIR}
+-        BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --config $<CONFIG> --target jpeg-static
+-        CMAKE_ARGS ${UHDR_CMAKE_ARGS} -DENABLE_SHARED=0
+-        BUILD_BYPRODUCTS ${JPEG_LIBRARIES}
+-        INSTALL_COMMAND ""
+-    )
+-  endif()
++if(CONAN_USE_JPEGTURBO)
++  find_package(libjpeg-turbo REQUIRED)
++  add_library(JPEG::JPEG ALIAS libjpeg-turbo::libjpeg-turbo)
++endif()
++if(CONAN_USE_MOZJPEG)
++  find_package(mozjpeg REQUIRED)
++  add_library(JPEG::JPEG ALIAS mozjpeg::mozjpeg)
+ endif()
+ 
+ if(UHDR_BUILD_JAVA)
+@@ -567,8 +523,8 @@ file(GLOB UHDR_TEST_SRCS_LIST "${TESTS_DIR}/*.cpp")
+ file(GLOB UHDR_BM_SRCS_LIST "${BENCHMARK_DIR}/*.cpp")
+ file(GLOB IMAGE_IO_SRCS_LIST "${THIRD_PARTY_DIR}/image_io/src/**/*.cc")
+ 
+-set(PRIVATE_INCLUDE_DIR ${SOURCE_DIR}/include/ ${JPEG_INCLUDE_DIRS})
+-set(COMMON_LIBS_LIST ${JPEG_LIBRARIES} Threads::Threads)
++set(PRIVATE_INCLUDE_DIR ${SOURCE_DIR}/include/)
++set(COMMON_LIBS_LIST JPEG::JPEG Threads::Threads)
+ 
+ ###########################################################
+ # Targets
+@@ -583,9 +539,9 @@ target_include_directories(${IMAGEIO_TARGET_NAME} PRIVATE
+ set(UHDR_CORE_LIB_NAME core)
+ add_library(${UHDR_CORE_LIB_NAME} STATIC ${UHDR_CORE_SRCS_LIST})
+ target_compile_options(${UHDR_CORE_LIB_NAME} PRIVATE ${UHDR_WERROR_FLAGS})
+-if(NOT JPEG_FOUND)
+-  add_dependencies(${UHDR_CORE_LIB_NAME} ${JPEGTURBO_TARGET_NAME})
+-endif()
++#if(NOT JPEG_FOUND)
++#  add_dependencies(${UHDR_CORE_LIB_NAME} ${JPEGTURBO_TARGET_NAME})
++#endif()
+ if(NOT MSVC)
+   target_compile_options(${UHDR_CORE_LIB_NAME} PRIVATE -Wall -Wextra -Wshadow)
+ endif()
+@@ -744,7 +700,7 @@ endif()
+ if(${CMAKE_SYSTEM_NAME} MATCHES "Android")
+   target_link_libraries(${UHDR_TARGET_NAME} PRIVATE ${log-lib})
+ endif()
+-target_link_libraries(${UHDR_TARGET_NAME} PRIVATE ${JPEG_LIBRARIES})
++target_link_libraries(${UHDR_TARGET_NAME} PRIVATE JPEG::JPEG)
+ set_target_properties(${UHDR_TARGET_NAME}
+                       PROPERTIES PUBLIC_HEADER ultrahdr_api.h)
+ if(BUILD_SHARED_LIBS)
+@@ -767,7 +723,7 @@ if(BUILD_SHARED_LIBS)
+   if(${CMAKE_SYSTEM_NAME} MATCHES "Android")
+     target_link_libraries(${UHDR_TARGET_NAME_STATIC} PRIVATE ${log-lib})
+   endif()
+-  target_link_libraries(${UHDR_TARGET_NAME_STATIC} PRIVATE ${JPEG_LIBRARIES})
++  target_link_libraries(${UHDR_TARGET_NAME_STATIC} PRIVATE JPEG::JPEG)
+   combine_static_libs(${UHDR_CORE_LIB_NAME} ${UHDR_TARGET_NAME_STATIC})
+   if(NOT MSVC)
+     set_target_properties(${UHDR_TARGET_NAME_STATIC}
+@@ -789,7 +745,8 @@ if(UHDR_BUILD_JAVA)
+ endif()
+ 
+ if(UHDR_ENABLE_INSTALL)
+-  if(NOT(MSVC OR XCODE))
++  if(TRUE)
++  #if(NOT(MSVC OR XCODE))
```

Comment:
```suggestion
-  if(NOT(MSVC OR XCODE))
+  if(TRUE)
```

This is fine, we can already see the `-` line 

However, we probably need a little context justifying this?